src/app/components/preview/preview.component.spec.ts

```diff
@@ -382,7 +382,7 @@ describe('PreviewComponent', () => {
     expect(router.navigate).toHaveBeenCalledWith(['personal-files', 'folder1']);
   });
 
-  it('should navigate to original location in case of Alfresco API errors', async () => {
+  xit('should navigate to original location in case of Alfresco API errors', async () => {
```

Comment:
this test is no longer failing after top test was changed by Bogdan... so the 'x' can be removed now. Is it ok that all these tests are outside a `describe`?